app/routes/admin/sales/locations.js

```diff
@@ -4,23 +4,13 @@ export default Route.extend({
   titleToken() {
     return this.get('l10n').t('Location');
   },
-  model() {
-    return [{
-      location         : 'Delhi',
-      completedTickets : 1,
-      completedSales   : 0.00,
-      placedTickets    : 1,
-      placedSales      : 0.00,
-      pendingTickets   : 1,
-      pendingSales     : 0.00
-    }, {
-      location         : 'Bangalore',
-      completedTickets : 3,
-      completedSales   : 23.30,
-      placedTickets    : 4,
-      placedSales      : 32.00,
-      pendingTickets   : 4,
-      pendingSales     : 32.00
-    }];
+
+  async model() {
+    let filterOptions = [];
+
+    return this.store.query('admin-sales-by-location', {
+      filter       : filterOptions,
+      'page[size]' : 10
```

Comment:
Don't limit the page size to 10, rather for now define it as 0 to fetch all the records as admin sales table does not has pagination yet. If you like, you can work on converting it to an ember model table after this PR is merged.